auth0_to_hubspot/program.py

```diff
@@ -0,0 +1,49 @@
+"""This program adds new Auth0 users to HubSpot as contacts."""
+
+import datetime
+import os
+
+from autokitteh.auth0 import auth0_client
+from autokitteh.hubspot import hubspot_client
+from hubspot.crm.contacts import SimplePublicObjectInput
+
+auth0 = auth0_client("auth0_conn")
+hubspot = hubspot_client("hubspot_conn")
+
+LOOKUP_HOURS = int(os.getenv("HOURS"))
+
+
+def check_for_new_users(event):
+    """Workflow entrypoint.
+    Looks up new Auth0 users in the last `HOURS` hours and adds them to HubSpot as contacts.
+    """
+    time_range = _get_time_range(LOOKUP_HOURS)
+    query = f"created_at:[{time_range['start']} TO {time_range['end']}]"
+
+    response = auth0.users.list(q=query, search_engine="v3")
+    add_new_users(response["users"])
+
+
+def _get_time_range(hours):
+    """Calculate start and end times for user lookup."""
+    now = datetime.datetime.utcnow()
+    start_time = now - datetime.timedelta(hours=hours)
+
+    return {"start": start_time.isoformat() + "Z", "end": now.isoformat() + "Z"}
```

Comment:
It's nice that "start" and "end" are explicit, but I think the query template in line 21 is clear enough ("X to Y") and using a dictionary only makes it look more complicated than it is.

A better and more common approach: return a tuple:

```
start, end = _get_time_range(LOOKUP_HOURS)
```

Reply:
Nice! Good suggestion. Done.